src/migrations/20190906054214-create-users.js

```diff
@@ -0,0 +1,65 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    username: {
+      type: Sequelize.STRING
+    },
+    firstName: {
+      type: Sequelize.STRING
+    },
+    lastName: {
+      type: Sequelize.STRING
+    },
+    email: {
+      type: Sequelize.STRING
+    },
+    password: {
+      type: Sequelize.STRING
+    },
+    isVerified: {
+      type: Sequelize.BOOLEAN
+    },
+    roleId: {
+      type: Sequelize.INTEGER
+    },
+    gender: {
+      type: Sequelize.STRING
+    },
+    dateOfBirth: {
+      type: Sequelize.DATE
+    },
+    preferredLanguage: {
+      type: Sequelize.STRING
+    },
+    preferredCurrency: {
+      type: Sequelize.STRING
+    },
+    residetialLocation: {
+      type: Sequelize.STRING
+    },
+    department: {
+      type: Sequelize.STRING
```

Comment:
Which field is department ? I'm not aware of this field

Reply:
This is part of the users' field of the users' table.